api/helpers/event.ts

```diff
@@ -0,0 +1,180 @@
+/**
+ * @file This file contains helper functions for handling Graph events.
+ */
+import { ethers } from "ethers";
+
+import { WinningTicketRedeemedEvent, MessageData } from "../types";
+
+const pricePerPixel = 0.0000000000000012; // (1200 wei)
```

Comment:
`pricePerPixel` was hardcoded based on the price the Studio gateway node was willing to pay at the time I wrote this script. We should update this to whatever that is today. Ideally, we fetch the orchestrator's price from another API. We _do_ have the [orchestrator price API]([url](https://github.com/livepeer/orchestrator-price-api)) which we could use to fetch the orchestrator's current advertised price. One thing to keep in mind if we use that endpoint doesn't store historical data, but that should be fine since this bot is for posting real time data.

Reply:
Fixed in https://github.com/adamsoffer/livepeer-payout-bot/pull/10.